friends/friends.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Friends Quiz</title>
+</head>
+<body>
+    
+
+<div class="main--content">
+
+
+    <h2 class="question">Question Comes Here</h2>
+
+    <ul>
+      <li>
+        <button type="radio" class="showbtn"><input type="radio" name="answers" id="ans1" class="answer"/>
+        <label for="ans1" id="opt--1">Option 1 </label></button>
+      </li>
+      <li>
+        <button class="showbtn"><input type="radio" name="answers" id="ans2" class="answer"/>
+        <label for="ans2" id="opt--2">Option 2</label></button>
+      </li>
+      <li>
+        <button class="showbtn"><input type="radio" name="answers" id="ans3" class="answer"/>
+        <label for="ans3" id="opt--3">Option 3 </label></button>
+      </li>
+      <li>
+       <button class="showbtn"><input type="radio" name="answers" id="ans4" class="answer"/>
+        <label for="ans4" id="opt--4">Option 4 </label></button> 
+      </li>
+    </ul>
+    <button id="submitBtn">Submit</button>
+</div>   
+ 
+
+<script src="/friends/friends.js"></script>
+</body>
+</html>
+
+
+
+
```

Comment:
remove such extra white space, and do practise using code formatter before push

Reply:
done